### PR TITLE
Fix some typos.

### PR DESCRIPTION
**Pull request by miketaylr**

### CODE REVIEW

**Comment by miketaylr** [comment]
> **ipr ** — The repository manager doesn't know the following repository: WICG/private-network-access

y tho

**Comment by miketaylr** [comment]
PTAL @letitz

File: index.src.html [hunk 1]
```diff
@@ -267,7 +267,7 @@ <h4 id="shortlinks">Navigation</h4>
     hosted at a <a>public address</a> in order to ensure that employees can work
     even when they're not at the office. How considerate!
 
-    Clicking `http://go/*` links from `https://mail.mega.corp/` will trigger a
+    Clicking `https://go/*` links from `https://mail.mega.corp/` will trigger a
```
Comment (mikewest) [inline index.src.html:270]: Yes, let's land this, even though it isn't true!

**Review by mikewest — APPROVED** [review]

**Comment by mikewest** [comment]
This PR is purely editorial, so I'm going to land it despite the failing check, but /cc @yoavweiss for the IPR check.

**Comment by yoavweiss** [comment]
Agree that this is not substantive. I also think @miketaylr is already a WICG member, so the main question is why are the bots borked..

**Comment by miketaylr** [comment]
> I also think @miketaylr is already a WICG member, so the main question is why are the bots borked..

Yep, I'm a WICG member. Also publishing seems to be busted (assuming it's automated), https://wicg.github.io/private-network-access/ is still showing June 23rd. 🤷

**Comment by yoavweiss** [comment]
> @yoavweiss halp! I don't know why the bots are borked. Where can I find out more?

Is this happening on other PRs as well?

**Comment by letitz** [comment]
Not that I know of, but there has been no other PR affecting the spec text since June 23rd. Would it help if I tried to merge a quick PR as a test?

**Comment by yoavweiss** [comment]
Yeah, that can help figure out if this is a one off or a persistent issue

**Comment by letitz** [comment]
Oh! Oh. :facepalm: 

The problem is not busted bots. The problem is that `make` was not run so `index.html` was not updated to reflect `index.src.html`. Sorry for the noise. I'll fix it.

**Comment by letitz** [comment]
Pushed 497ee7ec6cfddb8b5e572a8cfed4bcb8abeeb226.

**Pull request merged**